### ISSUE TITLE
Allow users to edit account info on Account page

### ISSUE THREAD

**Issue by whatSocks** [RadicalxChange/rxc-voice#41]
example:

* someone sends me some credits + an invite to a particular email
* I click the link and create an account using that email, and pick video verification 
* I try to log in again and see this message:
```
Unverified account

Sorry, but you can't access the site until you verify your account.

You can do that by clicking the link in your invitation email, creating an account, and logging into a valid Github or Twitter account to verify.
```
* Since I have already created an account, this message is prompting me to create another account with a different email and picking github or twitter as verification

I could do this, but then I'd lose my credits? Do I have to now go back to whoever invited me to ask for the credits to be sent to another email so I can pick the correct verification? Perhaps the person who sent me credits has already run out and I'm out of luck here?

**Comment by alexrandaccio** [comment]
Thanks for pointing this out, I need to update the language of this message to reflect the changes in #37 (added a new account verification option).

Let me explain some things for context:

**create account page**
- when you "invite" someone by sending them credits, an "unverified" user object is created for that person, and an invitation email is sent to their email address.
- the "create account" link in their invitation email contains a token that gives them permission to modify that user object. So when they click the link and fill out their information, this does not create a new user object. It modifies the one that was created automatically when the transfer was sent.
- this means that you can click the "create account" link and fill out the form as many times as you want, but you will only be modifying the same user object each time. No new user objects will be created.

**verifying accounts**
- as mentioned above, when a new user object is created, `user.is_verified = False` by default.
- when you attempt to login to an unverified account, you see the message that you posted in your comment.
- an account only gets marked as verified when one of the following happens:
  - the user successfully connects a Github or Twitter account
  **or**
  - a site admin marks the user as verified manually (after identifying them in a video call)


##TL;DR 
- you are seeing this message because you chose to verify your account by "Video application", and your application has not been approved yet (so you are still unverified)
- if you want to use the Github or Twitter option instead, you can simply click the link in your invitation email and fill out the form again. It will overwrite your account information with the new info and you won't lose any voice credits.

##changes I will make as a result of this issue
- update the `Sorry, but you can't access the site until you verify your account.` message to explain to users who chose the video application option why they can't login
- across the board, make it clearer to users how they can edit their account information

**Comment by whatSocks** [comment]
@alexrandaccio another confusing thing about the current flow 

* email is sent to a person (name@domain.com)
* they use the signup flow and pick a better email (name@domain2.com)
* they go to the account page -> see name@domain.com

it would be a lot easier to have an edit button within the app if the user wants to change something about their account. It could even take them to the same signup form, but having to hunt down an email to edit a field is very intuitive, especially since the changes don't take effect until the user re-authorizes.

**Comment by alexrandaccio** [comment]
> it would be a lot easier to have an edit button within the app if the user wants to change something about their account.

my thoughts exactly

**Comment by whatSocks** [comment]
Something that would help mitigate this usability problem would be to have labels on the fields so it's clear that what the user is seeing is a username not an email, and to add email to the view.
